bot/backend/apis/music.py

```diff
@@ -0,0 +1,24 @@
+"""interacting with the music api"""
+
+import tekore
+
+from decouple import config
+
+
+class MusicClient:
```

Comment:
The other API clients in the `backend/api` package all implement the `AbstractAPIClient` ABC (but I realize that most of the methods it enforces aren't necessary here as we're using tekore)
What's your opinion? (re: do we maintain uniformity within the `apis` package by implementing the ABC here too / just move this file out of `apis` / uniformity doesn't matter, this setup is fine)

Reply:
I think we can definitely rename the methods though (i.e `MusicClient` gets music data is already clear from it's name, so calling the method as `MusicClient.fetch_music_data` is a little redundant)

Reply:
I think this set up is fine. The apis folder is meant for files that interact with apis (I assume), so it makes sense to keep it there even if it doesn't require the ABC